data-types/datetime64/README.md

```diff
@@ -0,0 +1,33 @@
+# Datetime64 data type
+
+Defines a data type for a datetime object based on a 64-bit integer.
+
+## Permitted fill values
+
+The value of the `fill_value` metadata key must be a signed 64-bit integer.
```

Comment:
If we want to stick with the integer representation, we should clarify that this is a JSON representation of an integer between -2^63 and 2^63 - 1, inclusive. But as this data type is totally new for zarr v3 and we don't have to worry about backwards compatibility in the json format w.r.t. v2, we could consider a more date-like human-readable alternative to an int. I'd love to get feedback from heavy datetime users on what they would expect.